wiki/People/en.md

```diff
@@ -1,13 +1,12 @@
 # People
 
-The following are user groups, made up of the osu!community members that help maintain _osu!_
-Most of these user groups are given forum name colours, in-game chat colours, and/or profile badges.
+The following are user groups, made up of the osu!community members that help maintain osu! Most of these user groups are given forum name colours, in-game chat colours, and/or profile badges or titles.
```

Comment:
`in-game chat colours, and/or profile badges or titles.` -> `in-game chat colours, profile badges, and/or titles.`